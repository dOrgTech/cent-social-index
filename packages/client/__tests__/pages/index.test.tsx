import { render } from '@testing-library/react'
import Home from '@/pages/index'

describe('Home', () => {
  it('matches the snapshot', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
  })
})
