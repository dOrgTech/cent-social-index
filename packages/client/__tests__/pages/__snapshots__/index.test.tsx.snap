// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home matches the snapshot 1`] = `
<div>
  <div
    class="bg-white"
  >
    <nav
      class="bg-gray-800"
    >
      <div
        class="max-w-7xl mx-auto px-2 sm:px-6 lg:px-8"
      >
        <div
          class="relative flex items-center justify-between h-16"
        >
          <div
            class="absolute inset-y-0 left-0 flex items-center hidden"
          >
            <button
              aria-controls="mobile-menu"
              aria-expanded="false"
              class="inline-flex items-center justify-center p-2 rounded-md text-gray-400 hover:text-white hover:bg-gray-700 focus:outline-none focus:ring-2 focus:ring-inset focus:ring-white"
              type="button"
            >
              <span
                class="sr-only"
              >
                Open main menu
              </span>
              <svg
                aria-hidden="true"
                class="block h-6 w-6"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M4 6h16M4 12h16M4 18h16"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              <svg
                aria-hidden="true"
                class="hidden h-6 w-6"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M6 18L18 6M6 6l12 12"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </button>
          </div>
          <div
            class="flex-1 flex items-center justify-center"
          >
            <div
              class="flex-shrink-0 flex items-center"
            >
              <a
                href="https://cent.app.bio/"
              >
                <span
                  style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
                >
                  <span
                    style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                  >
                    <img
                      alt=""
                      aria-hidden="true"
                      src="data:image/svg+xml,%3csvg xmlns=%27http://www.w3.org/2000/svg%27 version=%271.1%27 width=%2740%27 height=%2740%27/%3e"
                      style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                    />
                  </span>
                  <img
                    alt="logo"
                    class="rounded-[50px]"
                    data-nimg="intrinsic"
                    decoding="async"
                    src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                    style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                  />
                  <noscript />
                </span>
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="sm:hidden"
        id="mobile-menu"
      />
    </nav>
    <div
      class="flex justify-center align-middle pt-4 pb-4"
    >
      <form
        class="w-full max-w-sm"
      >
        <div
          class="md:flex md:items-center"
        >
          <div
            class="md:w-1/3"
          >
            <label
              class="block text-gray-500 font-bold md:text-right mb-1 md:mb-0 pr-4"
              form="inline-full-name"
            >
              Eth Address
            </label>
          </div>
          <div
            class="md:w-2/3"
          >
            <input
              class="bg-gray-200 appearance-none border-2 border-gray-200 rounded w-full py-2 px-4 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-purple-800"
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          class="md:flex md:items-center"
        >
          <div
            class="md:w-1/3"
          />
          <span
            class="h-4"
          />
        </div>
        <div
          class="md:flex md:items-center mt-4"
        >
          <div
            class="md:w-1/3"
          />
          <div
            class="md:w-2/3"
          >
            <button
              class="shadow bg-purple-800 hover:bg-purple-700 focus:shadow-outline focus:outline-none text-white font-bold py-2 px-4 rounded"
              type="submit"
            >
              Get Ranks
            </button>
          </div>
        </div>
      </form>
    </div>
    <div
      class="flex justify-center align-middle pt-4 pb-4"
    >
      <div>
        <div
          class="invisible"
          id="d3-chart-wrapper"
        />
      </div>
    </div>
    <div
      class="flex sm:flex-col md:flex-row justify-center align-middle p-2"
    />
  </div>
</div>
`;
